README.md

```diff
@@ -136,6 +136,10 @@ chmod a+x .git/hooks/pre-commit
 You can also setup the [IntelliJ plugin](https://plugins.jetbrains.com/plugin/8527)
 to reformat code from within your IDE
 
+### Lombok
+This project uses [Lombok](https://projectlombok.org/) to reduce boilerplate. You can setup
+ the [IntelliJ plugin](https://plugins.jetbrains.com/plugin/6317) to add IDE support. 
```

Comment:
ps a lot of spring projects explicitly stopped using lombok due to complaints about IDEs etc. That doesn't mean shouldn't be done, just saying that in the end many spring cloud projects undid lombok.